sites/feedback/ajax/page-feedback-en.html

```diff
@@ -0,0 +1,84 @@
+<div id="gc-pft" class="row wb-disable-allow" data-wb-jsonmanager='{
+	"name": "gc-pft",
+	"extractor": [
+		{ "selector": "title", "path": "pageData/pageTitle" },
+		{ "selector": "html", "attr": "lang", "path": "pageData/language" },
+		{ "interface": "locationHref", "path": "pageData/submissionPage" },
+		{ "selector": "#wb-lng ul li:first-child a[lang]", "attr": "href", "path": "pageData/oppositelang" },
+		{ "selector": "[data-feedback-theme]", "attr": "data-feedback-theme", "path": "pageData/themeopt" },
+		{ "selector": "[data-feedback-section]", "attr": "data-feedback-section", "path": "pageData/sectionopt" },
+		{ "selector": "meta[name=\"dcterms.creator\"]", "attr": "content", "path": "pageData/institutionopt" },
+		{ "selector": "[data-feedback-link]", "attr": "data-feedback-link", "path": "contact/link" },
+		{ "selector": "[data-feedback-url]", "attr": "data-feedback-url", "path": "contact/url" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0">
```

Comment:
Comment/question

We should probably include in our CSS file for the pft all the style that are not means to be customized by implementer. At the same time it will increase the readability of the markup.

For example we can apply the following style directly in our CSS and not relying on the utilities class: "well", "margin", "padding", "text". 

@Garneauma @GormFrank what do you think?

Reply:
Let's keep it as is for now and we can think about it later on how we can improve the readability of the ajax fragment for the feedback in the situation when a department would like to create their own personalized feedback form.